### PR TITLE
updated cassandra, neo4j, elasticsearch backup roles

### PR DESCRIPTION
**Pull request by G33tha**

### CODE REVIEW

File: ansible/es_backup.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- hosts: "{{ remote }}"
+- hosts: "{{ groups['composite-search-cluster'][0] }}"
```
Comment (rjshrjndrn) [inline ansible/es_backup.yml]: replace the `groups['composite-search-cluster'][0]` with group_name, and provice `run_once: true`

File: ansible/roles/cassandra-backup/vars/main.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 cassandra_root_dir: /etc/cassandra
 cassandra_backup_dir: /data/cassandra/backup
+cassandra_backup_azure_container_name: lp-cassandra-backup
```
Comment (rjshrjndrn) [inline ansible/roles/cassandra-backup/vars/main.yml]: why are we having a vars/main.yml file ? please move it to defaults/main.yml

File: pipelines/backup/elasticsearch-restore/Jenkinsfile [hunk 3]
```diff
@@ -20,7 +20,7 @@ node() {
                 jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
                 currentWs = sh(returnStdout: true, script: 'pwd').trim()
                 ansiblePlaybook = "${currentWs}/ansible/es_restore.yml"
-                ansibleExtraArgs = "--extra-vars \"host=${params.host} SNAPSHOT_NUMBER=${params.SNAPSHOT_NUMBER}\""
+                ansibleExtraArgs = "--extra-vars \"es_restore_host=${params.host} SNAPSHOT_NUMBER=${params.SNAPSHOT_NUMBER}\""
```
Comment (rjshrjndrn) [inline pipelines/backup/elasticsearch-restore/Jenkinsfile]: do we have to pass this parms.host ?

File: pipelines/backup/neo4j-restore/Jenkinsfile [hunk 4]
```diff
@@ -20,7 +20,7 @@ node() {
                 jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
                 currentWs = sh(returnStdout: true, script: 'pwd').trim()
                 ansiblePlaybook = "${currentWs}/ansible/neo4j_restore.yml"
-                ansibleExtraArgs = "--extra-vars \"hosts=${params.hosts} neo4j_restore_gzip_file_name=${params.neo4j_restore_gzip_file_name} neo4j_restore_azure_storage_account_name=${params.neo4j_restore_azure_storage_account_name} neo4j_restore_azure_storage_access_key=${params.neo4j_restore_azure_storage_access_key} neo4j_restore_azure_container_name=${params.neo4j_restore_azure_container_name}\""
+                ansibleExtraArgs = "--extra-vars \"hosts=${params.hosts} neo4j_restore_gzip_file_name=${params.neo4j_restore_gzip_file_name}\""
```
Comment (rjshrjndrn) [inline pipelines/backup/neo4j-restore/Jenkinsfile]: do we have to pass params.host ?

Comment (G33tha) [reply pipelines/backup/neo4j-restore/Jenkinsfile]: yes we have

Comment (rjshrjndrn) [reply pipelines/backup/neo4j-restore/Jenkinsfile]: why's that ?

**Pull request merged**